src/index.js

```diff
@@ -0,0 +1,59 @@
+import '@fortawesome/fontawesome-free/js/fontawesome';
+import '@fortawesome/fontawesome-free/js/solid';
+import '@fortawesome/fontawesome-free/js/regular';
+import '@fortawesome/fontawesome-free/js/brands';
+import './main.scss';
+import domManager from './js/info';
+
+const APIKEY = 'eaed0043eb661ba7a7f56160196f9ad9';
+const btn = document.querySelector('.search__btn');
+const search = document.getElementById('search__input');
+const error = document.querySelector('.error');
+
+async function getWeather(location, unit) {
```

Comment:
You ought to use Es6 syntax as can be seen from this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)